gdp-src-build/conf/templates/renesas-rcar-gen3.local.inc

```diff
@@ -154,5 +154,8 @@ DISTRO_FEATURES_append = " avb"
 # Configuration for ivi-shell and ivi-extension
 #DISTRO_FEATURES_append = " ivi-shell"
 
+# Configuration for USB 3.0
```

Comment:
Why also here?   Is it correct understanding that this is supported on Salvator-x boards but not on the starter-kits?

Reply:
I decided to put it there also for a few reasons:

1. It can be considered as part of the standard R-Car Gen 3 "block" of entries for local.conf as found in the Salvator-X [example](https://github.com/renesas-rcar/meta-renesas/blob/00f70f062aace04c051fa92d3cd7b887718fc313/meta-rcar-gen3/docs/sample/conf/salvator-x/poky-gcc/mmp/local-wayland.conf#L386) from the YBSP. Having it there makes maintenance easier when checking for changes in YBSP updates.

2. Whilst it will not be enabled for the Starter Kits on their own as they have no USB 3.0, it is useful if a Kingfisher expansion board is attached which does.

3. Expanding this in-house to an A or B-Sample which may have USB 3.0 is easier.